src/main/scala/com/wixpress/common/specs2/JMock.scala

```diff
@@ -10,9 +10,9 @@ import org.specs2.execute._
 import org.specs2.main.{ArgumentsArgs, ArgumentsShortcuts}
 import org.specs2.matcher.{Expectations => _, _}
 import org.specs2.mock.HamcrestMatcherAdapter
-import org.specs2.mutable.Specification
 import org.specs2.specification.AroundEach
 import org.specs2.specification.core.SpecificationStructure
+import scala.language.experimental.macros
```

Comment:
I guess this won't be forward compatible with scala 3, since macros are removed there

Reply:
Scala 3 has `inline` which will allow the same functionality - with much saner syntax.
But yes it is not compatible.
It is possible to configure sbt to have different sources for the same classes based on scala version